src/superphot_plus/fit_numpyro.py

```diff
@@ -735,6 +846,17 @@ def jax_model(
 
 
 def flux_from_posteriors(t, params, max_flux):
+    """Generates green and red band fluxes (TODOLIV - does this make sense?) from given posteriors.
```

Comment:
Weirdly enough, g- and r- bands are NOT optically green and red (more specifically the g-band center wavelength is closer to blue). I would just call it 'g- and r-band fluxes'

Reply:
Oh wow, thanks - I'll get on that!